WikiTemplate/iOS/wikitemplate-ios.md

```diff
@@ -155,3 +190,10 @@
 ## Session storage
 
 - [ ] Verify that tabs restore when closed, including active tab
+
+## Navigation [iPad]
```

Comment:
Suggestion

Maybe good to move this below the Navigation section above

Reply:
Yeah makes sense, I can change it. I initially placed it on the end because it would be easier to exclude when copying.